deployed-contracts/isolated-pools.md

```diff
@@ -67,6 +69,18 @@
   * vUSDD\_Tron: [`0xf1da185CCe5BeD1BeBbb3007Ef738Ea4224025F7`](https://bscscan.com/address/0xf1da185CCe5BeD1BeBbb3007Ef738Ea4224025F7)
   * vUSDT\_Tron: [`0x281E5378f99A4bc55b295ABc0A3E7eD32Deba059`](https://bscscan.com/address/0x281E5378f99A4bc55b295ABc0A3E7eD32Deba059)
 
+### Token Converters
+
+* RiskFundConverter: [`0xA5622D276CcbB8d9BBE3D1ffd1BB11a0032E53F0`](https://bscscan.com/address/0xA5622D276CcbB8d9BBE3D1ffd1BB11a0032E53F0)
+* XVSVaultTreasury: [`0x269ff7818DB317f60E386D2be0B259e1a324a40a`](https://bscscan.com/address/0x269ff7818DB317f60E386D2be0B259e1a324a40a)
+* SingleTokenConverterBeacon: [`0x4c9D57b05B245c40235D720A5f3A592f3DfF11ca`](https://bscscan.com/address/0x4c9D57b05B245c40235D720A5f3A592f3DfF11ca)
+* USDTPrimeConverter: [`0xD9f101AA67F3D72662609a2703387242452078C3`](https://bscscan.com/address/0xD9f101AA67F3D72662609a2703387242452078C3)
+* USDCPrimeConverter: [`0xa758c9C215B6c4198F0a0e3FA46395Fa15Db691b`](https://bscscan.com/address/0xa758c9C215B6c4198F0a0e3FA46395Fa15Db691b)
+* BTCBPrimeConverter: [`0xE8CeAa79f082768f99266dFd208d665d2Dd18f53`](https://bscscan.com/address/0xE8CeAa79f082768f99266dFd208d665d2Dd18f53)
+* ETHPrimeConverter: [`0xca430B8A97Ea918fF634162acb0b731445B8195E`](https://bscscan.com/address/0xca430B8A97Ea918fF634162acb0b731445B8195E)
+* XVSVaultConverter: [`0xd5b9AE835F4C59272032B3B954417179573331E0`](https://bscscan.com/address/0xd5b9AE835F4C59272032B3B954417179573331E0)
+* ConverterNetwork: [`0xF7Caad5CeB0209165f2dFE71c92aDe14d0F15995`](https://bscscan.com/address/0xF7Caad5CeB0209165f2dFE71c92aDe14d0F15995)
+
```

Comment:
I would move these addresses to their own file: `/deployed-contracts/token-converters.md`. They are not used only by the IL pools. I would move to this new file also the testnet addresses, and I would update the SUMMARY file to add the new link to the page

Reply:
https://github.com/VenusProtocol/venus-protocol-documentation/pull/70/commits/ce5560e2b1b7bd809b7cc1513dbe81c5af3eb0b4